test_utilities/src/res.rs

```diff
@@ -45,7 +44,7 @@ fn setup_values_dir<P: AsRef<Path>, L: AsRef<str>>(res_path: P, locale_id: Optio
 
 fn setup_strings_file<P: AsRef<Path>, L: AsRef<str>>(
     locale_values_dir_path: P,
-    locale_id: Option<L>,
+    _locale_id: Option<L>,
```

Comment:
where is `_locale_id` used?

Reply:
Nowhere :).. I didn't do clippy for test files & so these occurrences escaped me. There are a few more such warnings in test sources. Will fix this along with that in another PR & update the CI run accordingly. Sound good?